inject/src/main/java/io/avaje/inject/event/Event.java

```diff
@@ -37,9 +37,16 @@
 public abstract class Event<T> {
 
   protected final List<Observer<T>> observers;
+  protected final String defaultQualifier;
```

Comment:
Should this be renamed to `qualifier`?  Seems like `defaultQualifier` isn't really a default per se?